File: movefiles.py
import os
import shutil

base_dir = r"C:\Users\Alkaeya De la Pena\Documents\bootcamp\boiler-plate-nextjs"
components_dir = os.path.join(base_dir, "components")

# Create components folder
os.makedirs(components_dir, exist_ok=True)
print("✓ Created components folder")

# Files to move
files = [
    'Navbar.tsx',
    'HeroSection.tsx',
    'AboutSection.tsx',
    'SkillsSection.tsx',
    'ExperienceSection.tsx',
    'EducationSection.tsx',
    'ProjectsSection.tsx',
    'ContactSection.tsx',
    'Footer.tsx'
]

# Move files
for file in files:
    source_path = os.path.join(base_dir, file)
    dest_path = os.path.join(components_dir, file)
    
    if os.path.exists(source_path):
        shutil.move(source_path, dest_path)
        print(f"✓ Moved {file}")
    else:
        print(f"✗ File not found: {file}")

# List contents of components folder
print("\n📁 Components folder contents:")
contents = os.listdir(components_dir)
for file in sorted(contents):
    print(f"   - {file}")

print(f"\n✓ Total files in components folder: {len(contents)}")

File: movefiles.js
const fs = require('fs');
const path = require('path');
const { execSync } = require('child_process');

const baseDir = "C:\\Users\\Alkaeya De la Pena\\Documents\\bootcamp\\boiler-plate-nextjs";
const componentsDir = path.join(baseDir, "components");

// Create components folder
if (!fs.existsSync(componentsDir)) {
    fs.mkdirSync(componentsDir, { recursive: true });
    console.log("✓ Created components folder");
}

// Files to move
const files = [
    'Navbar.tsx',
    'HeroSection.tsx',
    'AboutSection.tsx',
    'SkillsSection.tsx',
    'ExperienceSection.tsx',
    'EducationSection.tsx',
    'ProjectsSection.tsx',
    'ContactSection.tsx',
    'Footer.tsx'
];

// Move files
files.forEach(file => {
    const sourcePath = path.join(baseDir, file);
    const destPath = path.join(componentsDir, file);
    
    if (fs.existsSync(sourcePath)) {
        fs.renameSync(sourcePath, destPath);
        console.log(`✓ Moved ${file}`);
    } else {
        console.log(`✗ File not found: ${file}`);
    }
});

// List contents of components folder
console.log("\n📁 Components folder contents:");
const contents = fs.readdirSync(componentsDir);
contents.forEach(file => {
    console.log(`   - ${file}`);
});

console.log(`\n✓ Total files in components folder: ${contents.length}`);
